src/pages/components/checkbox.mdx

```diff
@@ -1,15 +1,183 @@
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+A Checkbox’s label is part of its touch target.
+
+import Default from '../../examples/forms/checkbox/Default.tsx';
+import DefaultCode from '!!raw-loader!../../examples/forms/checkbox/Default.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Regular weight label
+
+Checkbox labels are emphasized by default but can be regular weight.
+
+import RegularWeightLabel from '../../examples/forms/checkbox/RegularWeightLabel.tsx';
+import RegularWeightLabelCode from '!!raw-loader!../../examples/forms/checkbox/RegularWeightLabel.tsx';
+
+<CodeExample code={RegularWeightLabelCode}>
+  <RegularWeightLabel />
+</CodeExample>
+
+### Indeterminate
+
+If a checkbox has child checkboxes, it should show an indeterminate state (reflected by a dash) when
+the child checkboxes contains both some checked and unchecked checkboxes.
+
+import Indeterminate from '../../examples/forms/checkbox/Indeterminate.tsx';
+import IndeterminateCode from '!!raw-loader!../../examples/forms/checkbox/Indeterminate.tsx';
+
+<CodeExample code={IndeterminateCode}>
+  <Indeterminate />
+</CodeExample>
+
+### Hint text
+
+Hint adds additional context to the Checkbox label.
+
+import Hint from '../../examples/forms/checkbox/Hint.tsx';
+import HintCode from '!!raw-loader!../../examples/forms/checkbox/Hint.tsx';
+
+<CodeExample code={HintCode}>
+  <Hint />
+</CodeExample>
+
+### Validation
+
+Success, warning, or danger validation messages are shown using the Message component with the
+Checkbox.
+
+import Validation from '../../examples/forms/checkbox/Validation.tsx';
+import ValidationCode from '!!raw-loader!../../examples/forms/checkbox/Validation.tsx';
+
+<CodeExample code={ValidationCode}>
+  <Validation />
+</CodeExample>
+
+### Hidden
+
+Checkbox labels can be hidden.
+
+import Hidden from '../../examples/forms/checkbox/Hidden.tsx';
+import HiddenCode from '!!raw-loader!../../examples/forms/checkbox/Hidden.tsx';
+
+<CodeExample code={HiddenCode}>
+  <Hidden />
+</CodeExample>
+
+### Disabled
+
+Disabled Checkboxes are not interactive, and not perceived by a screen reader.
+
+import Disabled from '../../examples/forms/checkbox/Disabled.tsx';
+import DisabledCode from '!!raw-loader!../../examples/forms/checkbox/Disabled.tsx';
+
+<CodeExample code={DisabledCode}>
+  <Disabled />
+</CodeExample>
+
+## How to use it well
+
+### Use to select any number of options
+
+<BestPractice>
+  <Do imageSource={props.data.checkboxSelectionDo.children[0].publicURL}>
+    <p>Use to select multiple options from a set of two or more.</p>
+    <p>
+      Use to turn a standalone setting on and off, as a favorable alternative to a{' '}
+      <a href="./toggle">Toggle</a>.
+    </p>
+  </Do>
+  <Dont imageSource={props.data.checkboxStandaloneOptionDont.children[0].publicURL}>
+    <p>
+      Use to select exactly one option from a set of two or more. Use <a href="../radio">Radios</a>
+      &nbsp;instead.
```

Comment:
Please check the deployment. Now both `../radio` links are broken. Looks like we've used `/components/radio` style links in other locations. Also, seems we typically use `{' '}` rather than `&nbsp;` 🤷 

/cc @rossmoody

Reply:
Oh, is this a manual fix because of the issue I referenced on SplitButtons with Prettier and MarkdownLint? If you update the url to "/components/radio" you might get lucky and have it break in a spot that passes.

Reply:
For posterity, I wasn't adding {' '}, Prettier was adding it for me on `yarn format`

Reply:
A lint rule requires each line to be 100 characters maximum. Adjusting this sentence to have a 100 character limit gets the linting to pass during the commit hooks, but during the formatting phase, Prettier will insert `{'  '}` which pushes the character limit beyond 100.

Reply:
Sounds like the linting config should be tweaked.

Reply:
Just tested. The fact that `.prettierrc` and `.markdownlintrc` are equal to each other seems to be a problem. I'd recommend setting `.markdownlintrc` to `"line-length: 120"`.

Reply:
Awesome, sounds good! I'll update the linting config instead of manually adding `&nbsp;`.

Reply:
Awesome, sounds good! I'll update the linting config instead of manually adding `&nbsp;`.

Edit: Updated `.markdownlintrc` to `"line-length: 120"` so now Prettier rules play well with Markdown lint rules.